Update index.md (#4351)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/install/configuration/http-proxy/index.md

              "gatekeeper.d2iq.com/mutate": "pod-proxy"
    ```

1.  You can create a `kommander` namespace, or the namespace where Kommander will be installed, and then label it such that the Gatekeeper mutation is active on the namespace.

    ```bash
    kubectl create namespace kommander
    kubectl label namespace kommander gatekeeper.d2iq.com/mutate=pod-proxy
    ```

1.  Install Kommander using the above configuration file:

>     <p class="message--note"><strong>NOTE: </strong>An alternative to initializing the KUBECONFIG environment variable (by running <code>export KUBECONFIG=cluster_name.conf</code>) is to use the <code>--kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander is installed on the workload cluster.</p>

Reviewer: ```suggestion
    <p class="message--note"><strong>NOTE: </strong>To ensure Kommander is installed on the workload cluster, use the <code>--kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander is installed on the workload cluster.</p>
```
